distributed/cli/dask_scheduler.py

```diff
@@ -110,6 +122,8 @@ def del_pid_file():
         scheduler.stop()
         if bokeh_proc:
             bokeh_proc.close()
+        if os.path.exists(local_directory):
```

Comment:
It sounds like we should review what we do there generally.  You could probably avoid this issue by following @pitrou 's suggestion rather than mine and simply remove the directory of the file after importing it.